src/main/java/org/elasticsearch/search/aggregations/bucket/significant/heuristics/ScriptHeuristic.java

```diff
@@ -109,6 +113,7 @@ public double getScore(long subsetFreq, long subsetSize, long supersetFreq, long
             ESLoggerFactory.getLogger("script heuristic").warn("cannot compute score - script has not been initialized yet.");
             return 0;
         }
+        script.setNextVar("_term", term);
```

Comment:
@brwe This is the one bit of this change I'm not sure about. Unlike the other Long fields passed to this script via final LongAccessors the "_term" variable can either be a Long or a String so I call setNextVar here with each scoring call. Your LongAccessor objects only need to be registered once using setNextVar on initialization. I'm not sure if my repeated calls to setNextVar will be less efficient here.